src/BuildIt.Forms/BuildIt.Forms.Controls/ContentButton.xaml.cs

```diff
@@ -223,6 +223,13 @@ private void OnCommandChanged()
                 if (IsPressed)
                 {
                     await VisualStateManager.GoToState(this, "Pressed");
+
+                    // Releasing the button after half a second. This should stop the button's visual state getting "stuck" in a ListView -RR
+                    await Task.Delay(500);
+                    if (VisualStateManager.GetVisualStateGroups(this)?.StateManager?.CurrentState("CommonStates") == "Pressed")
+                    {
+                        OnTouchEffectAction(this, new TouchActionEventArgs(0, TouchActionType.Released, Point.Zero, false));
```

Comment:
What happens with a button not in a list - if the user holds down the button, do we raise two Released events?

Reply:
If the user holds down the button it will auto-release after 0.5 seconds, no matter what the context of the button is. It's definitely not a perfect solution, but since there is no logic/event/command that fires in a content button when it is in the "Pressed" state, it shouldn't matter too much.
I tested it in Yambay, and didn't observe two "Released" events being fired, because line-167 `                        if (IsEntered || !EnterRequired)` stops the pressed event from being fired in a Released-to-Released transition.